frontend/package.json

```diff
@@ -18,11 +21,16 @@
     "@types/bootstrap": "^5.2.5",
     "@types/node": "^16.11.56",
     "@vitejs/plugin-vue": "^3.0.3",
+    "@vitejs/plugin-vue-jsx": "^2.0.1",
```

Comment:
Hi Basil, would you mind sync with 'main' branch again?
I think this '-jsx' is already been removed from main so we can get rid of this one.

Reply:
I did merge main, this change must have gotten missed when I merged.

Reply:
Pushing a change to address this now.